web/params.go

```diff
@@ -236,6 +237,12 @@ func ParamsFromRequest(r *http.Request) *Params {
 		params.Page = val
 	}
 
+	if val, err := strconv.ParseInt(query.Get("timestamp"), 10, 64); err != nil || val < 0 {
+		params.Timestamp = model.GetMillis()
```

Comment:
This can lead on a not easy to detect bug around timesmap, for example, passing a wrongly formatted timestamp gets automatically converted in to "now"

Reply:
Mkay. I'll add strict checking